UPGRADE.rst

```diff
@@ -105,6 +105,29 @@ shown below:
 
           return {"localpart": localpart}
 
+Removal historically Synapse Admin API 
+--------------------------------------
+
+Historically, the Synapse Admin API has been accessible under:
+
+* ``/_matrix/client/api/v1/admin``
+* ``/_matrix/client/unstable/admin``
+* ``/_matrix/client/r0/admin``
+* ``/_synapse/admin/v1``
+
+These endpoints with ``/_matrix/client/*`` prefixes have been removed.
+From this point, the Admin API will only be accessible under:
+
+* ``/_synapse/admin/v1``
+
+The only exception is:
+
+* ``/_matrix/client/r0/admin/whois/<user_id>``
+* ``/_synapse/admin/v1/whois/<user_id>``
```

Comment:
I don't think we need to provide the full paths again (since there is potentially multiple paths this is available at on the client-server API):

```suggestion
The only exception is the `/admin/whois` endpoint, which is
`also available via the client-server API <https://matrix.org/docs/spec/client_server/r0.6.1#get-matrix-client-r0-admin-whois-userid>`_.
```